src/main/java/uk/gov/hmcts/reform/sscs/config/NotificationEventTypeLists.java

```diff
@@ -75,12 +76,12 @@ private NotificationEventTypeLists() {
         PERMISSION_TO_APPEAL_GRANTED,
         PERMISSION_TO_APPEAL_REFUSED,
         PERMISSION_TO_APPEAL_REQUEST,
-        BUNDLE_CREATED_FOR_UPPER_TRIBUNAL,
-        LIBERTY_TO_APPLY_REQUEST
```

Comment:
Is this LTA constant meant to be removed here?

Reply:
Yes, as it was a duplicate